geometry_msgs/Pose origin
std_msgs/Float64 height
std_msgs/Float64 radius
---
std_msgs/Bool status
